apps/io.exoworlds

```diff
@@ -0,0 +1 @@
+
```

Comment:
{
    "name": "ExoWorlds",
    "href": "www.exoworlds.io",
    "desc": "10K Exoplanets & Systems. Build Civilizations, Join the Galactic Senate & Explore.",
    "tags": ["NFT","ExoWorlds","VIP-181","Collectibles","VNFT"]
}